app/models/activity.rb

```diff
@@ -212,4 +214,8 @@ def requires_call_dates?
   def forecasted_total_for_date_range(range:)
     planned_disbursements.where(period_start_date: range).sum(:value)
   end
+
+  def requires_intended_beneficiaries?
+    recipient_region? || (recipient_country? && intended_beneficiaries != "Not Applicable")
```

Comment:
Isn't `intended_beneficiaries` an array? As I've alluded to above, this would be more cleanly handled by adding a `boolean` field to the model to indicate whether intended beneficiaries are required, and then wiring the `Not applicable` checkbox up to that field.

Reply:
Not sure I fully understand your suggestion. We can pair on that on our 1:1 tomorrow if that's okay?